docs/src/staking/README.md

```diff
@@ -0,0 +1,116 @@
+# Staking on Solana
+
+Staking your SOL tokens on Solana is the best way you can help secure the world's
+highest-performing blockchain network, and
+[earn rewards](../implemented-proposals/staking-rewards.md) for doing so!
+
+Solana is a Delegated Proof-of-Stake system (DPoS), which means that anyone who
+holds SOL tokens can choose to delegate some of their SOL to one or more
+validators, who process transactions and run the network.
+
+Delegating stake is a shared-risk shared-reward financial model that can provide
```

Comment:
You'll want to run this whole paragraph by Jeff. While it's a nice intuition, the association may be imprecise and have legal implications.